test/Microsoft.ML.StaticPipelineTesting/Training.cs

```diff
@@ -47,8 +47,9 @@ public void SdcaRegression()
             LinearRegressionModelParameters pred = null;
 
             var est = reader.MakeNewEstimator()
-                .Append(r => (r.label, score: ctx.Trainers.Sdca(r.label, r.features, maxIterations: 2,
-                onFit: p => pred = p, advancedSettings: s => s.NumThreads = 1)));
+                .Append(r => (r.label, score: ctx.Trainers.Sdca(r.label, r.features, null,
+                new SdcaRegressionTrainer.Options() { MaxIterations = 2, NumThreads = 1 },
```

Comment:
>NumThreads [](start = 73, length = 10)

@ if numthreads should be 1, would it better to default NumThreads in options to 1?

Reply:
modifying default settings is out of purview of this PR.

i think we are using NumThreads = 1  only for test cases to be more stable

---
In reply to: [248875274](https://github.com/dotnet/machinelearning/pull/2093#discussion_r248875274) [](ancestors = 248875274)